CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [User] When detaching a default filter select, reset the default filter as default (PR [#5321](https://github.com/vatesfr/xen-orchestra/pull/5321))
```

Comment:
If this change only avoid storing `null` values in the DB, it will not be visible by users, so, the entry in the CHANGELOG will not be necessary.